systems/registry/member/github.com/ukama/ukama/systems/registry/network/pb/gen/network_grpc.pb.go

```diff
@@ -0,0 +1,399 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
We need to fix the proto gen path so that generated files are place in the the right pb/ sub-folder  instead of created a new github.com/... tree.

Reply:
I think this is something checked in by mistake. I dont see these genrated files at all.